cli/templatepush.go

```diff
@@ -188,10 +241,15 @@ func (r *RootCmd) templatePush() *clibase.Cmd {
 				return err
 			}
 
+			if utf8.RuneCountInString(name) > 31 {
```

Comment:
Nit:
```suggestion
			if utf8.RuneCountInString(name) >= 32 {
```

less cognitive overhead to read the if statement and comment this way